_pkgdown.yml

```diff
@@ -3,6 +3,18 @@ url: https://mlr-org.github.io/mlr
 template:
   params:
     bootswatch: cosmo
+  path: pkgdown/templates
+  assets: pkgdown/assets
+  default_assets: false
+  docsearch:
```

Comment:
The docsearch info should be *under* params, not at the same level:

```yaml
template:
  params:
    docsearch:
      api_key: e300ecafdf04fe1199e3339c825ce7d0
      index_name: mlr

```

Reply:
Thanks!

Reply:
HTH. I think you'll also have to update your custom templates to get the search bar.